[CAS-948] Global Customization Documentation (#1835)
GetStream/stream-chat-android: documentation/docs/ui/global_customizations/chat_ui_customizations/url_signer.md

---
id: uiGlobalCustomizationsUrlSigner
title: UrlSigner
sidebar_position: 3
---

> It is possible to use a custom url signer to `ChatUi` by adding a new `UrlSigner` implementing the following interface:

Reviewer: ```suggestion
`ChatUi` allows to use of a custom url signer by implementing the following interface:
```

```kotlin
public interface UrlSigner {
    public fun signFileUrl(url: String): String
    public fun signImageUrl(url: String): String
}
```

> This is the way to  add a new UrlSigner:

Reviewer: ```suggestion
This is the way to add a new `UrlSigner`:
```
